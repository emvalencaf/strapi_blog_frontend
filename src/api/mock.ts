import { ResponseLoadPosts, ResponseRequest } from "./types";

export const mockResponseRequest = {
	setting: {
		data: {
			id: "1",
			attributes: {
				blogName: "lorem ipsum blog",
				blogDescription: "lorem ipsum blog",
				menuLink: [
					{
						id: "1",
						link: "#link1",
						text: "link1",
						newTab: false,
					},
					{
						id: "2",
						link: "#link2",
						text: "link2",
						newTab: false,
					},
				],
				logo: {
					data: {
						attributes: {
							alternativeText: null,
							url: "https://res.cloudinary.com/dnk5clykr/image/upload/v1673403271/coding_logo_design_template_vector_70ae4b4683.jpg",
						},
					},
				},
				footer: "<p>lorem ipsum footer</p>",
			},
		},
	},
	posts: {
		data: [
			{
				id: "1",
				attributes: {
					title: "Lorem ipsum",
					slug: "lorem-ipsum",
					excerpt:
						"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer imperdiet cursus diam, et hendrerit tortor suscipit at. Praesent quis felis turpis. Nullam a neque ante. Praesent scelerisque pharetra.",
					content:
						'<h2>HTML Ipsum Presents</h2><p><strong>Pellentesque habitant morbi tristique</strong> senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. <i>Aenean ultricies mi vitae est.</i> Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, commodo vitae, ornare sit amet, wisi. Aenean fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus lacus enim ac dui. <a href="#">Donec non enim</a> in turpis pulvinar facilisis. Ut felis.</p><h2>Header Level 2</h2><ol><li>Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</li><li>Aliquam tincidunt mauris eu risus.</li></ol><blockquote><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus magna. Cras in mi at felis aliquet congue. Ut a est eget ligula molestie gravida. Curabitur massa. Donec eleifend, libero at sagittis mollis, tellus est malesuada tellus, at luctus turpis elit sit amet quam. Vivamus pretium ornare est.</p></blockquote><h3>Header Level 3</h3><ul><li>Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</li><li>Aliquam tincidunt mauris eu risus.</li></ul><p>\n\t\t\t\t#header h1 a {\n\t\t\t\t &nbsp;display: block;\n\t\t\t\t &nbsp;width: 300px;\n\t\t\t\t &nbsp;height: 80px;\n\t\t\t\t}\n\t\t\t\t</p>',
					allowComments: true,
					cover: {
						data: {
							id: "9",
							attributes: {
								alternativeText: null,
								url: "https://res.cloudinary.com/dnk5clykr/image/upload/v1673378509/what_is_coding_1024x683_b9743d63c6.jpg",
							},
						},
					},
					categories: {
						data: [
							{
								id: "1",
								attributes: {
									displayName: "javascript",
									slug: "javascript",
								},
							},
						],
					},
					tags: {
						data: [
							{
								id: "3",
								attributes: {
									displayName: "frontend",
									slug: "frontend",
								},
							},
							{
								id: "1",
								attributes: {
									displayName: "reactjs",
									slug: "reactjs",
								},
							},
							{
								id: "2",
								attributes: {
									displayName: "nextjs",
									slug: "nextjs",
								},
							},
						],
					},
					author: {
						data: {
							id: "1",
							attributes: {
								displayName: "Edson Valença",
								slug: "edson-valenca",
							},
						},
					},
				},
			},
		],
	},
} as ResponseRequest;

export const mockResponseLoadPost = {
	settings: {
		id: "1",

		blogName: "lorem ipsum blog",
		blogDescription: "lorem ipsum blog",
		menuLink: [
			{
				id: "1",
				link: "#link1",
				text: "link1",
				newTab: false,
			},
			{
				id: "2",
				link: "#link2",
				text: "link2",
				newTab: false,
			},
		],
		logo: {
			id: null,
			alternativeText: null,
			url: "https://res.cloudinary.com/dnk5clykr/image/upload/v1673403271/coding_logo_design_template_vector_70ae4b4683.jpg",
		},
		footer: "<p>lorem ipsum footer</p>",
	},
	posts: [
		{
			id: "1",

			title: "Lorem ipsum",
			slug: "lorem-ipsum",
			excerpt:
				"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer imperdiet cursus diam, et hendrerit tortor suscipit at. Praesent quis felis turpis. Nullam a neque ante. Praesent scelerisque pharetra.",
			content:
				'<h2>HTML Ipsum Presents</h2><p><strong>Pellentesque habitant morbi tristique</strong> senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. <i>Aenean ultricies mi vitae est.</i> Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, commodo vitae, ornare sit amet, wisi. Aenean fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus lacus enim ac dui. <a href="#">Donec non enim</a> in turpis pulvinar facilisis. Ut felis.</p><h2>Header Level 2</h2><ol><li>Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</li><li>Aliquam tincidunt mauris eu risus.</li></ol><blockquote><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus magna. Cras in mi at felis aliquet congue. Ut a est eget ligula molestie gravida. Curabitur massa. Donec eleifend, libero at sagittis mollis, tellus est malesuada tellus, at luctus turpis elit sit amet quam. Vivamus pretium ornare est.</p></blockquote><h3>Header Level 3</h3><ul><li>Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</li><li>Aliquam tincidunt mauris eu risus.</li></ul><p>\n\t\t\t\t#header h1 a {\n\t\t\t\t &nbsp;display: block;\n\t\t\t\t &nbsp;width: 300px;\n\t\t\t\t &nbsp;height: 80px;\n\t\t\t\t}\n\t\t\t\t</p>',
			allowComments: true,
			cover: {
				id: "9",

				alternativeText: null,
				url: "https://res.cloudinary.com/dnk5clykr/image/upload/v1673378509/what_is_coding_1024x683_b9743d63c6.jpg",
			},
			categories: [
				{
					id: "1",

					displayName: "javascript",
					slug: "javascript",
				},
			],

			tags: [
				{
					id: "3",

					displayName: "frontend",
					slug: "frontend",
				},
				{
					id: "1",

					displayName: "reactjs",
					slug: "reactjs",
				},
				{
					id: "2",

					displayName: "nextjs",
					slug: "nextjs",
				},
			],

			author: {
				id: "1",

				displayName: "Edson Valença",
				slug: "edson-valenca",
			},
		},
	],
} as ResponseLoadPosts;
