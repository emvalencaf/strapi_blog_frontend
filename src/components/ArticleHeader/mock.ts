import { ArticleHeaderProps } from ".";

export default {
	title: "lorem ipsum title",
	excerpt: "lorem ipsum lorem impsum",
	cover: {
		url: "image.jpg",
	},
	createdAt: "2021-03-02T15:16:09.246+00:00",
	author: {
		id: "woek2131239481",
		slug: "lorem-ipsum",
		displayName: "Lorem ipsum",
	},
	categories: [
		{
			id: "ijmwerw19212",
			displayName: "Tech",
			slug: "tech",
		},
		{
			id: "wiejj293888923",
			displayName: "Javascript",
			slug: "javascript",
		},
	],
} as ArticleHeaderProps;
