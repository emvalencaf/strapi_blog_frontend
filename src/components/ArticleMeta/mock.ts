import { ArticleMetaProps } from ".";

export default {
	createdAt: "2021-03-02T15:16:09.246+00:00",
	author: {
		id: "woek2131239481",
		slug: "lorem-ipsum",
		displayName: "Lorem ipsum",
	},
	categories: [
		{
			id: "ijmwerw19212",
			displayName: "Tech",
			slug: "tech",
		},
		{
			id: "wiejj293888923",
			displayName: "Javascript",
			slug: "javascript",
		},
	],
} as ArticleMetaProps;
