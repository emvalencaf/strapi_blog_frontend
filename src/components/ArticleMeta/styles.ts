import styled, { css } from "styled-components";

export const Wrapper = styled.footer`
	${({ theme }) => css`
		font-size: ${theme.font.sizes.small};
		color: ${theme.colors.darkerGray};

		.categories span::after {
			content: ", ";
		}

		.categories span:last-child::after {
			content: "";
		}

		a {
			color: ${theme.colors.secondary};
			text-decoration: none;
			transition: : all 350ms ease-in-out;

			&:hover {
				filter: brightness(50%);
			}
		}


	`}
`;
