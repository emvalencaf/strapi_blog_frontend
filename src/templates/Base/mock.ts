import { data } from "../../api/data.json";
// types
import { BaseTemplateProps } from ".";

export default {
	settings: data.setting,
	children: "Ol√° mundo",
} as BaseTemplateProps;
