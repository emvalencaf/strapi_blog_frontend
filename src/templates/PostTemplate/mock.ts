import { data } from "../../api/data.json";
// type
import { PostTemplateProps } from ".";

export default {
	settings: data.setting,
	post: data.posts[0],
} as PostTemplateProps;
