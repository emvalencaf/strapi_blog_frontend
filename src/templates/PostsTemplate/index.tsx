// template
import BaseTemplate from "../Base";

// components
import PostGrid from "../../components/PostGrid";

// types
import { PostStrapi } from "../../shared-typed/post-strapi";
import { SettingsStrapi } from "../../shared-typed/settings-strapi";
export type PostsTemplateProps = {
	settings: SettingsStrapi;
	posts?: PostStrapi[];
};

const PostsTemplate = ({ settings, posts = [] }: PostsTemplateProps) => {
	return (
		<BaseTemplate settings={settings}>
			<PostGrid posts={posts} />
		</BaseTemplate>
	);
};

export default PostsTemplate;
