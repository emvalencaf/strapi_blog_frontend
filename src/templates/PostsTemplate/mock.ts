import { data } from "../../api/data.json";

// type
import { PostsTemplateProps } from ".";

export default {
	settings: data.setting,
	posts: data.posts,
} as PostsTemplateProps;
