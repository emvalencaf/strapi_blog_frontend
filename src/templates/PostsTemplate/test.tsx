import { screen } from "@testing-library/react";
import { renderTheme } from "../../styles/render-theme";

// react component to be test
import PostsTemplate from ".";

// mock
import mock from "./mock";

describe("<PostsTemplate />", () => {
	it("should render a PostsTempalte without posts", () => {
		renderTheme(<PostsTemplate {...mock} posts={undefined} />);

		expect(screen.getByText("Nenhum post encontrado")).toBeInTheDocument();
	});
	it("should match a snapshot", () => {
		const { container } = renderTheme(<PostsTemplate {...mock} />);

		expect(container).toMatchSnapshot();
	});
});
