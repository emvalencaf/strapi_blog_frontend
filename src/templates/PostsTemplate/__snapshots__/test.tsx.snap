// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostsTemplate /> should match a snapshot 1`] = `
.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
}

.c7 > img {
  width: 15rem;
  height: 15rem;
  border-radius: 50%;
}

.c6 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c16 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: none;
}

.c8 {
  display: block;
  color: #FFFFFF;
  -webkit-text-decoration: none;
  text-decoration: none;
  margin-bottom: 1.6rem;
  font-size: 1.8rem;
  border-right: 0.5rem solid #000000;
  -webkit-transition: all 350ms ease-in-out;
  transition: all 350ms ease-in-out;
}

.c8:hover {
  border-right: 0.5rem solid #dc143c;
  color: #dc143c;
}

.c2 {
  background: #000000;
  padding: 3.2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: fixed;
  z-index: 1;
  width: 100%;
  max-width: 32rem;
  height: 100vh;
  top: 0;
  left: 0;
  -webkit-transition: all 350ms ease-in-out;
  transition: all 350ms ease-in-out;
  left: -30rem;
  overflow-y: hidden;
}

.c3 {
  margin: auto;
  width: 100%;
}

.c4 .c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0;
  margin-bottom: 4.0rem;
}

.c4 .c5 img {
  border: 0.5rem solid #dc143c;
}

.c0 {
  position: fixed;
  top: 2.4rem;
  color: #FFFFFF;
  background: #000000;
  z-index: 2;
  width: 3rem;
  height: 3rem;
  left: 26rem;
  -webkit-transition: all 350ms ease-in-out;
  transition: all 350ms ease-in-out;
  left: -1rem;
  color: #FFFFFF;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding-bottom: 6.4rem;
  width: 100%;
  max-width: 96rem;
  color: #333333;
  font-size: 1.6rem;
  margin: 0 auto;
}

.c10 .c5 {
  margin: 0 0 calc(1.6rem - 1rem);
}

.c20 {
  font-size: calc(1.6rem + 0.2rem);
  line-height: 1.5;
}

.c20 p {
  margin: 2.4rem 0;
}

.c20 a,
.c20 a:visited,
.c20 a:link {
  color: #dc143c;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 350ms ease-in-out;
  transition: all 350ms ease-in-out;
}

.c20 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c20 code {
  font-family: monospace;
  color: #dc143c;
  font-size: 1.6rem;
  background-color: #DDDDDD;
  padding: 0.2rem;
  margin: 0.2rem;
}

.c20 pre {
  font-size: 1.6rem;
  background-color: #000000;
  padding: 2.4rem;
  font-family: monospace;
  color: #FFFFFF margin:2.4rem 0;
  width: 100%;
  overflow-x: auto;
  font-size: 1.6rem;
}

.c20 pre code {
  color: inherit;
  background: inherit;
}

.c20 img {
  max-width: 100%;
}

.c20 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 2.4rem 0;
}

.c20 .image figcaption {
  font-size: 1.6rem;
  padding: 1.6rem;
  text-align: center;
  line-height: 1.3;
}

.c20 .image-style-side {
  max-width: 50%;
  float: right;
  margin: 2.4rem 0;
}

.c20 hr {
  border: none;
  border-bottom: 0.1rem solid $#DDDDDD;
}

.c20 ul,
.c20 ol {
  margin: 2.4rem 4.0rem;
}

.c20 .table {
  width: 100%;
  overflow: hidden;
  overflow-x: auto;
}

.c20 table {
  width: 100%;
  border-collapse: collapse;
  margin: 2.4rem 0;
}

.c20 .table td,
.c20 .table th {
  padding: 1.6rem;
  border: 0.1rem solid #DDDDDD;
}

.c20 blockquote {
  border-left: 0.5rem solid #dc143c;
  color: #AAAAAA;
  -webkit-filter: brightness(80%);
  filter: brightness(80%);
  padding-left: 2.4rem;
  font-style: italic;
  margin: 2.4rem;
}

.c18 {
  text-align: center;
  border-top: 0.1rem solid #DDDDDD;
}

.c18 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1.6rem;
}

.c18 .c19 {
  font-size: 1.6rem;
}

.c9 {
  margin-top: 6.4rem;
}

.c11 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  margin-bottom: 6.4rem;
}

.c17 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  padding: 3.2rem;
}

.c21 {
  position: fixed;
  background: #000000;
  color: #FFFFFF;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
}

.c12 {
  width: 100%;
}

.c13 {
  max-width: 120rem;
  margin: 0 auto;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(28rem,1fr));
  gap: 3.2rem;
  padding: 3.2rem;
}

.c14 .c5 {
  margin-left: 0;
  margin-right: 0;
  margin-top: calc(1.6rem - 1.4rem);
  margin-bottom: calc(1.6rem - 1rem);
}

.c14 a {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  -webkit-transition: all 350ms ease-in-out;
  transition: all 350ms ease-in-out;
}

.c14:hover a {
  color: #dc143c;
}

.c14:hover img {
  opacity: 0.8;
}

.c15 {
  max-width: 100%;
  -webkit-transition: opacity 350ms ease-in-out;
  transition: opacity 350ms ease-in-out;
}

@media (max-width:768px) {
  .c2 {
    left: -32rem;
  }
}

@media (max-width:768px) {
  .c0 {
    left: 1rem;
  }
}

@media (max-width:768px) {
  .c20 {
    font-size: 2rem;
  }

  .c20 .image-style-side {
    max-width: 100%;
    float: none;
    margin: 0;
  }
}

@media (max-width:360px) {
  .c13 {
    grid-template-columns: 1fr;
  }
}

<div>
  <div
    class=""
  >
    <a
      aria-label="Open or close menu"
      class="c0"
      href="#"
      title="Open or close menu"
    >
      <svg
        aria-hidden="true"
        aria-label="Open menu"
        class="c1"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
        />
      </svg>
    </a>
    <div
      aria-hidden="true"
      class="c2"
    >
      <nav
        class="c3"
      >
        <div
          class="c4"
        >
          <h1
            class="c5 c6"
          >
            <a
              class="c7"
              href="/"
              target="_self"
            >
              <img
                alt="Otávio Miranda"
                src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614698965/Ubuntu_gel_by_Midge_Mantissa_Sinnaeve_7caea7ddb4.jpg"
              />
            </a>
          </h1>
        </div>
        <a
          class="c8"
          href="#target"
          target="_self"
        >
          O texto
        </a>
        <a
          class="c8"
          href="#link2"
          target="_blank"
        >
          o LINK 2
        </a>
      </nav>
    </div>
    <div
      class="c9"
    >
      <header
        class="c10"
      >
        <h1
          class="c5 c6"
        >
          <a
            class="c7"
            href="/"
            target="_self"
          >
            <img
              alt="Otávio Miranda - Uma descrição para o seu blog."
              src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614698965/Ubuntu_gel_by_Midge_Mantissa_Sinnaeve_7caea7ddb4.jpg"
            />
          </a>
        </h1>
      </header>
    </div>
    <div
      class="c11"
    >
      <div
        class="c12"
      >
        <div
          class="c13"
        >
          <div
            class="c14"
          >
            <a
              href="/post/vis-nobis-veritus-maluisset-id-nonumy-adversarium"
            >
              <img
                alt="Vis nobis veritus maluisset id, nonumy adversarium"
                class="c15"
                src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614698965/Ubuntu_gel_by_Midge_Mantissa_Sinnaeve_7caea7ddb4.jpg"
              />
            </a>
            <h3
              class="c5 c16"
            >
              <a
                href="/post/vis-nobis-veritus-maluisset-id-nonumy-adversarium"
              >
                Vis nobis veritus maluisset id, nonumy adversarium
              </a>
            </h3>
            <p
              class=""
            >
              Fugit aliquip erroribus eu eam, an mel vidit doctus. Vis nobis veritus maluisset id, nonumy adversarium neglegentur est ut. Tation persius pri eu, modo minim fastidii mei no. Ad tota pertinax definiebas eam.
            </p>
          </div>
          <div
            class="c14"
          >
            <a
              href="/post/sea-array-destructuring-dicat-fabulas-consetetur"
            >
              <img
                alt="Sea Array Destructuring dicat fabulas consetetur "
                class="c15"
                src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614699042/Origin_of_nature_by_Julian_Tomasini_dcfdd17cbe.jpg"
              />
            </a>
            <h3
              class="c5 c16"
            >
              <a
                href="/post/sea-array-destructuring-dicat-fabulas-consetetur"
              >
                Sea Array Destructuring dicat fabulas consetetur 
              </a>
            </h3>
            <p
              class=""
            >
              Quod oblique mei ne, cetero delicatissimi sea cu, cu est harum dicit verterem. Tota forensibus ut usu, luptatum dignissim persecuti vel ex. Mea nostrud instructior cu, et referrentur theophrastus eum. Pro in iuvaret senserit.
            </p>
          </div>
          <div
            class="c14"
          >
            <a
              href="/post/quod-oblique-mei-ne-cetero-delicatissimi-sea"
            >
              <img
                alt="Quod oblique mei ne, cetero delicatissimi sea"
                class="c15"
                src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614699042/Origin_of_nature_by_Julian_Tomasini_dcfdd17cbe.jpg"
              />
            </a>
            <h3
              class="c5 c16"
            >
              <a
                href="/post/quod-oblique-mei-ne-cetero-delicatissimi-sea"
              >
                Quod oblique mei ne, cetero delicatissimi sea
              </a>
            </h3>
            <p
              class=""
            >
              Et sed legere rationibus, vitae evertitur ad eam. Eu amet vero officiis vis. Ad nullam pericula neglegentur sit, vix ei regione legendos argumentum. Mei an quas convenire, nobis dicunt mei ut. 
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c17"
    >
      <footer
        class="c18"
      >
        <div
          class="c19 c20"
        >
          <p>
            O texto da footer
          </p>
        </div>
      </footer>
    </div>
    <a
      aria-label="Go to top"
      class="c21"
      href="#"
      title="Go to top"
    >
      <svg
        aria-hidden="true"
        class="c1"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M7.41 15.41 12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
        />
      </svg>
    </a>
  </div>
</div>
`;
